### PR TITLE
[DNS] Internal feedback

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/dns/zone-setups/partial-setup/convert-partial-to-full.md [hunk 1]
```diff
@@ -24,11 +24,15 @@ Universal SSL only supports first-level subdomains. For more coverage and additi
...
 
-After deleting your zone activation TXT record, wait for its Time to Live (TTL) to expire.
+{{<Aside type="note">}}
 
-At least 24 hours prior to converting your application, disable DNSSEC.
+As a best practice, you should also delete the previous [zone activation TXT record](/dns/zone-setups/partial-setup/setup/#step-1--add-your-domain-to-cloudflare) at your authoritative DNS provider. To locate this value in the Cloudflare dashboard, go to **DNS** and find the **Verification TXT Record**.
+  
+{{</Aside>}}
+
+After deleting your zone activation TXT record, wait for its Time to Live (TTL) to expire.
```
Comment (kodster28) [inline content/dns/zone-setups/partial-setup/convert-partial-to-full.md]:
```suggestion
```

File: content/dns/zone-setups/partial-setup/convert-partial-to-full.md [hunk 2]
```diff
@@ -24,11 +24,14 @@ Universal SSL only supports first-level subdomains. For more coverage and additi
 
 ## Step 2 — Update settings in authoritative DNS
 
-At your authoritative DNS provider, you need to delete the previous [zone activation TXT record](/dns/zone-setups/partial-setup/setup/#step-1--add-your-domain-to-cloudflare). To locate this value in the Cloudflare dashboard, go to **DNS** and find the **Verification TXT Record**.
+At least 24 hours prior to converting your application, disable DNSSEC.
```
Comment (mmalden) [inline content/dns/zone-setups/partial-setup/convert-partial-to-full.md]:
```suggestion
At least 24 hours prior to converting your application, [disable DNSSEC](/dns/additional-options/dnssec/).
```

**Review by mmalden — APPROVED** [review]

File: content/dns/zone-setups/partial-setup/convert-partial-to-full.md (hunk 2 above)
Comment (kodster28) [reply content/dns/zone-setups/partial-setup/convert-partial-to-full.md]: Ah, let me update that to be clearer. It's not w/in CF.

Comment (kodster28) [inline content/dns/zone-setups/partial-setup/convert-partial-to-full.md]:
```suggestion
At least 24 hours prior to converting your application, disable DNSSEC at your authoritative DNS provider.
```

**Pull request merged**